### DISCUSSION TITLE
How to turn on PLJAVA extension

### DISCUSSION THREAD

**Discussion by mpahic** [supabase/supabase#1851] [Questions] [answered]
I am trying to enable PLJAVA extension on the Extensions page, but I get the following error:
Toggle PLJAVA failed: cannot use pl/java before successfully completing its setup

Is this a bug or must I do somethin before I can enable that?

**Comment by dragarcia** [comment, marked as answer]
Hi @mpahic! 👋 

There is indeed a bug. Fortunately, it can be fixed through your dashboard with the following SQL queries:
```sql
-- the bug was that the value for pljava.libjvm_location was incorrect
-- this command alters it to the correct one
alter system SET pljava.libjvm_location TO '/usr/lib/jvm/java-11-openjdk-arm64/lib/server/libjvm.so';

-- this reloads the configuration to reflect the change made above
select pg_reload_conf();

-- you should now be able to create the pljava extension
create extension pljava;
```

Do tell me if this works for you! Also, we'll be patching this bug in our next version of Supabase Postgres. In the meantime, this fix should suffice.

**Reply by mpahic** [reply]
Yes it works.
Thank you very much.

**Reply by burggraf** [reply]
Confirmed -- this works.  Thank you!

**Comment by wiverson** [comment]
Not working for me. 

1. First I run `alter system SET pljava.libjvm_location TO '/usr/lib/jvm/java-11-openjdk-arm64/lib/server/libjvm.so';' it returns "Success. No rows returned". 
2. Then I run `select pg_reload_conf();` it returns true. 
3. However, when I run `select * from pg_settings where name = 'pljava.libjvm_location'` immediately afterwards it returns no rows found. 
4. Next, running `create extension pljava;` returns Failed to run sql query: cannot use PL/Java before successfully completing its setup

Per another error message I created a `sqlj` schema with a simple create schema command.

Any more documentation on using PL/Java with Supabase? Ideally both how to turn it on and the recommended way to install the PL/Java jar?
